### DISCUSSION TITLE
Wallet adress changed on 1.2.0

### DISCUSSION THREAD

**Discussion by MaximusChia** [Chia-Network/chia-blockchain#7196] [General]
after installing the 1.2.0 update, my wallet adress has changed.
However all my balance are kept
Old wallet adress is not listed
Any explanation for this=?

**Comment by firmicus** [comment]
Guessing: First wallet address ??

**Reply by MaximusChia** [reply]
Yes, it was my first wallet adress, it stills displaying on config.yalm... why on gui is different?

**Reply by loppefaaret** [reply]
it is for privacy concern - the wallet detected the address had been used, and generated a new receive address - it is able to generated many many receive addresses, and they all lead back to your wallet!
